examples/SingleParticleTest/include/simulation_defines/unitless/physicalConstants.unitless

```diff
@@ -93,4 +93,10 @@ namespace picongpu
     // = 1/c^2
     const float_X MUE0_EPS0 = float_X(1. / SPEED_OF_LIGHT / SPEED_OF_LIGHT);
 
+    /* Atomic unit of electric field in PIC Efield units */
+    const float_X ATOMIC_UNIT_EFIELD = float_X(SI::ATOMIC_UNIT_EFIELD / UNIT_EFIELD);
+
+    /* Atomic unit of time in PIC units */
+    const float_X ATOMIC_UNIT_TIME = float_X(SI::ATOMIC_UNIT_TIME / UNIT_TIME);
```

Comment:
@ax3l : please take a look to the unit `ATOMIC_UNIT_TIME`. 
It is a 32 bit unit and not a 64 bit like all other units. The user convert the [SI](https://github.com/ComputationalRadiationPhysics/picongpu/pull/922/files#diff-49440648e4b4a4ad8969db4d104ec7c8R80) energies before to AU, therefore it looks like no `float32` overflow but please check it again.